fortran/src/H5Gff.F90

```diff
@@ -1180,7 +1180,7 @@ END SUBROUTINE h5gget_info_f90
 !! \param idx_type   Index type.
 !! \param order      Order of the count in the index.
 !! \param n          Position in the index of the group for which information is retrieved.
-!! \param ginfo      Derived type in which group information is returned
+!! \param ginfo      Derived type in which group information is returned.
```

Comment:
I didn't know there was a requirement for a '.' in parameter list...  It looks weird.

Reply:
I agree. Remove . if it is not a full sentence.